src/commands/mod.rs

```diff
@@ -1395,6 +1395,12 @@ pub struct ListVaultsEntry {
     pub moved_at: Option<u32>,
 }
 
+impl ListVaultsEntry {
+    pub fn outpoint(&self) -> OutPoint {
+        OutPoint::new(self.txid, self.vout)
+    }
+}
```

Comment:
Nothing is missing for a client of the API to do that on their own?

I mean i'm not against including it, it doesn't hurt. But then one probably wants that for something else as well, and another thing, etc.. While all the fields are `pub` and they can just use them.

Reply:
Ok I will remove it